### DISCUSSION TITLE
How to give space between pie chart and labels

### DISCUSSION THREAD

**Discussion by yudisyudistira12** [filamentphp/filament#11593] [Help]
### Package

Widgets

### Package Version

v.3.0

### How can we help you?

![Screenshot 2024-02-27 135942](https://github.com/filamentphp/filament/assets/59831982/2f281d38-ca7f-4841-976b-a793dd4f170a)

how do ensure that the position of the labels is not too close to the pie chart ?

**Comment by leandrocfe** [comment]
Can you share the code please?

**Reply by yudisyudistira12** [reply]
sure. 
![carbon](https://github.com/filamentphp/filament/assets/59831982/de8604a5-9104-40a7-bfa5-8de9ab82ef3e)


This my full code of the chart widget
